docs/setup.rst

```diff
@@ -22,6 +22,14 @@ The latest development version can also be installed directly from github.
 
    pip install git+https://github.com/credo-ai/credoai_lens#develop
 
+If you are interested in contributing to the development of Lens and
+would like to install requirements for testing and formatting, you'll
+have to install the dev requirements.
+
+::
+
+   pip install credoai-lens[dev]
```

Comment:
How will this impact using `pip install -e .` for getting the installed version of Lens to change depending on git branch?

Reply:
It won't change anything.  If you want to install locally with all dependencies you'd type `pip instlal -e ".[dev]"`. But that just affects the installation of dependencies. What you are talking about is referencing the local (changing) folder for the actual credoai library, which will be unchanged.